### PR TITLE
release 2021.04.12

### PR DESCRIPTION
**Pull request by goern**
the hread branch name is wrong... ;)

/assign @fridex 
/assign @harshad16

### CODE REVIEW

**Comment by goern** [comment]
/hole until we did the release on thoth-application

File: docs/TermsAndConditionsForTheScrum.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+* might have a due date assigned and confirmed by the team lead
+
+## Sprint Planning Session (experiment: integrate it with backlog refinement session)
+
+Pull the issues which support the high-level team objectives/theme and key results into the ‘next’ list, so that the
+team is clear of what is to be worked on in the next sprint.
+
+During the session, issues are open to allocation to any team member, and the team agrees to the interested team member
+to work on the issue. If the interested team member needs to be trained to fulfil the task, then a 'th
+task has to be added and taken by the team member. If the training task is not feasible, another team member should
```
Comment (harshad16) [inline docs/TermsAndConditionsForTheScrum.md:50]:
```suggestion
task has to be added for training required and taken by another team member who is knowledgeable of the topic. If the training task is not feasible, another team member should
```

File: RELEASES.md [hunk 2]
```diff
@@ -9,6 +9,30 @@ state of planning, have a look at our [Projects](https://github.com/orgs/thoth-s
 For all the nitty gritty details of our features (backend or services) have a look at our
 [Sprint Demo recordings](http://bit.ly/thoth-on-youtube).
 
+## Release 2021.04.12
+
+With this "Cumulative update Release" we announce the continouous update of all Thoth Services running on
```
Comment (harshad16) [inline RELEASES.md:14]:
```suggestion
With this "Cumulative update Release" we announce the continuous update of all Thoth Services running on
```

**Review by harshad16 — COMMENTED** [review]
Thanks this looks great
dropped few suggestions.
/lgtm

**Comment by fridex** [comment]
Nice! 👍🏻 
/lgtm

**Comment by goern** [comment]
> dropped few suggestions.

thanks, I shouldnt push in a hurry

**Comment by harshad16** [comment]
> /hold until we did the release on thoth-application

New Release made: https://github.com/thoth-station/thoth-application/releases/tag/v2021.04.12

**Pull request merged**